modules/dailymotionBidAdapter.md

```diff
@@ -0,0 +1,74 @@
+# Overview
+
+```
+Module Name: Dailymotion Bid Adapter
+Module Type: Bidder Adapter
+Maintainer: ad-leo-engineering@dailymotion.com
+```
+
+# Description
+
+Dailymotion prebid adapter.
+
+# Configuration options
+
+Before calling this adapter, you need to set its configuration with a call to setConfig like this:
+
+```javascript
+config.setConfig({
```

Comment:
Don't configure your adapter this way, see how other bidders define params on the ad unit, eg the appnexus md file, or using pbjs.setBidderConfig, eg on grid.

Reply:
Hello @patmmccann thank you for the feedback! Updated to use `setBidderConfig` on latest commit and updated documentation on https://github.com/prebid/prebid.github.io/pull/5091 as well